ddtrace/contrib/structlog/patch.py

```diff
@@ -79,6 +98,9 @@ def patch():
     if hasattr(structlog, "getLogger"):
         _w(structlog, "getLogger", _w_get_logger)
 
+    if hasattr(structlog, "configure"):
+        _w(structlog, "configure", _w_configure)
```

Comment:
I think we also need to hook [`reset_defaults`](https://github.com/hynek/structlog/blob/a62bbf6e41338b56f3a6e966d5fa00b0e0e929ea/src/structlog/_config.py#L290)